examples/server_hardware.go

```diff
@@ -16,7 +16,7 @@ func main() {
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		2000,
+		2200,
 		"*")
 
```

Comment:
Line 79 is using hardware type URI which is hardcoded

Reply:
done